src/index.js

```diff
@@ -350,20 +350,31 @@ class Backend {
             loadPath,
             i18nextElectronBackend
         } = this.backendOptions;
-        let filename = this.services.interpolator.interpolate(loadPath, {
+        const filename = this.services.interpolator.interpolate(loadPath, {
             lng: language,
             ns: namespace
         });
 
+        // Handle initImmediate option
```

Comment:
Can you explain a little bit more what this section does?

Reply:
Default value for initImmediate is true. This means resources are loaded asynchronously.
If user decides to set it to false, resources are loaded synchronously.
More info here : https://www.i18next.com/overview/configuration-options

Reply:
Great, thanks for explaining this to me, I was unsure where this came from. I see it must be necessary for loading translations in the main electron process now.